main.c

```diff
@@ -1061,10 +1062,16 @@ int main(int argc, char **argv)
 	fileidx = options->startnum < filecnt ? options->startnum : 0;
 
 	#if START_FROM_FILE_PATCH
-	if (options->startfile != NULL)
-		for (int i = 0; i < filecnt; ++i)
-			if (strcmp(options->startfile, files[i].path) == 0)
+	if (options->startfile != NULL) {
+		char startfile[PATH_MAX];
```

Comment:
[Don't use PATH_MAX](https://eklitzke.org/path-max-is-tricky), just use NULL and allow `realpath` to do the allocation as needed.

Reply:
Thanks, I've implemented that.